schema/deploy/shipping/views.sql

```diff
@@ -844,4 +844,95 @@ comment on view shipping.hcov19_observation_v1 is
   'Custom view of hCoV-19 samples with presence-absence results and best available encounter data';
 
 
+create or replace view shipping.scan_return_results_v1 as
+
+    with status_codes as (
+        -- Create status codes as required by Lab Med's results portal
+        with x(status_code, status) as (
+            values
+            ('DET', 'positive result, colored red + bold'),
+            ('INCON', 'inconclusive, colored orange; treated as positive with respect to information given'),
+            ('NDET', 'not detected/negative, colored blue'),
+            ('pending-other', null),
+            ('pending-lab', 'We are working on it!'),
+            ('cancelled', null),
+            ('notReceived', 'This is our code we made up')
```

Comment:
This code is not contained in the codes UW Lab Med sent us.

Reply:
If we can't have extra codes, I think we just want to merge this with `pending-lab`?

Reply:
`pending-lab` as I have it is our equivalent for `processing`. I'd ideally like to keep "processing" distinct from "not received".

Reply:
We could make it "pending-other", but I was planning on using "pending-other" as our internal sign that the test is positive but participant hasn't yet been contacted, so hold results.

Reply:
Yeah, just not sure we want to report `notReceived`...Well I think this also loops back to the language issue. No one has given us any guidance on the language to use for the return results site...

Reply:
We chatted on slack and decided to use `pending` (no distinction)